terraform/modules/credhub/variables.tf

```diff
@@ -17,6 +17,10 @@ variable "rds_parameter_group_name" {
   default = ""
 }
 
+variable "rds_parameter_group_family" {
+  default = "postgres9.6"
```

Comment:
Should we update our default to match version 12 now given all of the updates?  Or is this remaining because this DB hasn't been touched yet and needs to be handled separately?

Reply:
There's one more DB to go that looks like it should be a separate upgrade, but yes, we'll want to update the default to 12.4 after all updates have been applied.

Reply:
Totally fine, which is why we had all of these split in the first place!  I just wanted to make sure I remembered correctly and we weren't missing something else. :-)  Thanks, @jfredrickson5!